src/common/io.py

```diff
@@ -1218,6 +1218,13 @@ def __init__(self, fname, delayed_trajectory_loading = True):
                 all at the same time. Only works for files or streams that
                 are based on a file and has attribute name.
                 Default: True (for files)
+                
+            allow_file_updates --
+                If this is turned on, file updates (in terms of more
```

Comment:
I would rephrase "If this is turned on" to "If this is set to True" to clarify also that it is a boolean value.

Reply:
Fixed. However, I believe we should allow anything that is interpreted as True (in True Python spirit : ) )